integration-tests/src/test/java/org/zowe/apiml/functional/gateway/ValidateAPITest.java

```diff
@@ -89,6 +90,7 @@ void testGetEndpointNonConformant() {
 
     @Test
     @TestsNotMeantForZowe
+    @Disabled("skipping as its WIP in anothe PR")
```

Comment:
Related to #3623

Reply:
I've disabled the whole class in #3623

Reply:
Ok, I'll revert the changes to this file to avoid any conflicts